main.py

```diff
@@ -177,82 +122,120 @@ def noCode():
         pixels[noCode]=(0, 0, 0)
     pixels.show()
     time.sleep(0.3)
-#while True:
-    #noCode()
 
-def disabled():
-    for disabled in range(0, 15):
-        pixels[disabled]=(255, 0, 0)
+def RobotDisabled(current_mode):
```

Comment:
nit: python function names should start with lower case letters and be snake_case